README.md

```diff
@@ -26,7 +26,7 @@ Recent reports suggesting more clones of the D1 is now working as expected. This
 
 ### Connecting the components
 
-![Photo of schematics](Schematics/tag_reader_schematics_v2.png)
```

Comment:
I know this is not exactly your fault @rosscullen as the repo was already set this way. 
But there is no point in creating version numbers for files in git... You should just be overriding the old file with a new one. Git is the version manager. cc @adonno @magnusoverli